### ISSUE TITLE
[Feature Request] Always Show YouTube Progress Bar

### ISSUE THREAD

**Issue by shonay** [ParticleCore/Iridium#185]
Hi there,
Sorry I'm not sure if this is the correct place to write this. I was just wondering if it would be at all possible to include a feature in your script to have the YouTube Progress Bar to always be visible....
I used to use YouTube Always Show Progress Bar by Nemanja Bu ( https://greasyfork.org/en/scripts/30046-youtube-always-show-progress-bar ), however it seems as of today, the feature no longer works and I don't believe that the author/dev is going to be upgrading his script.

Basically all it does is, when the mouse is moved away from the video, it minimizes the bar to be a red line that progresses along the bottom of the video that's playing on YouTube thereby showing the progress of the video without us having to scroll the mouse back to the video to see how far along we are.

It's alright if you don't want to or cannot achieve this, just wanted to find out before giving up on the feature.

Thanks.

**Comment by ParticleCore** [comment]
Duplicate of https://github.com/ParticleCore/Iridium/issues/22
